.changeset/calm-dancers-accept.md

```diff
@@ -0,0 +1,9 @@
+---
+'@sveltejs/vite-plugin-svelte': major
+---
+
+change default value of compilerOptions.hydratable to false
+
+This is done to align with svelte compiler defaults and improve output in non-ssr scenarios.
+
+Add `{compilerOptions: {hydratable: true}}` to vite-plugin-svelte config if you need hydration (eg. for ssr)
```

Comment:
Should mention that defaults to true when Vite options.ssr is true, no? Or isn't this the case?

Reply:
not the case as not all SSR solutions set that and I chose consistency over automatic default that only works some of the time